ibis-server/app/model/__init__.py

```diff
@@ -112,7 +112,11 @@ class MySqlConnectionInfo(BaseModel):
     database: SecretStr
     user: SecretStr
     password: SecretStr | None = None
-    ssl_mode: SecretStr | None = Field(alias="sslMode", default=None)
+    ssl_mode: SecretStr | None = Field(
+        alias="sslMode",
+        default="ENABLED",
+        description="Use ssl connection or not. The default value is `ENABLED` because MySQL uses `caching_sha2_password` by default and the driver MySQLdb support caching_sha2_password with ssl only.",
+    )
```

Comment:
The ssl_model is enabled by default because ibis ports to MySQLdb that only support caching_sha2_password with SSL.
c.c @ongdisheng who implements this.